grep-00XX-relicense-volk.md

```diff
@@ -53,6 +53,100 @@ We consider it to be very important to let every contributor know that the wish
+get-go. The FSF recommendations on libraries states that "If developers are
+already using an established alternative library released under a nonfree
+license or a lax pushover license, then we recommend using the GNU Lesser
+General Public License (LGPL)."
+(https://www.gnu.org/licenses/license-recommendations.html).
+
+We agree with this recommendation, and want to avoid VOLK being a SIMD library
+exclusively used by GNU Radio (and a few others), while other software packages
+rally behind other SIMD libraries, that are more liberally licensed.
+
+We could have chosen an even less restrictive license, but we are believers in
+copyleft, and believe the LGPL strikes the right balance between not restricting
+users in which types of software they may use VOLK, and requiring users to
+continue upstreaming their modifications to VOLK.
+
+##### How are we going to do this?
+
+Relicensing codebases is a complex process, full of potential pitfalls. To avoid
+any misconceptions, ambiguities, or legal issues, we have decided to go for the
+nuclear option: We will delete all of VOLK, and start from scratch, using the
+LGPL from the beginning of VOLK 3.0. However, it would be a futile attempt to
+try and recreate the decades of work that went into previous versions of VOLK
+without violating the VOLK 2.0 license. We are therefore asking all contributors
+to re-grant us permission to include their code, previously submitted into VOLK
+versions 1 and 2, into the newly licensed VOLK 3. The process for contributors
+is simple (see below), but first, we want to explain how this works.
+
+For contributing to VOLK 2.0, most of you signed a CLA, granting the copyright
+to the FSF. This CLA includes a paragraph stating that the FSF grants back to
+the developer the rights to use their contributions as they see fit.
+
+We cannot copy code from VOLK 2.0, since that is GPL licensed. But we can ask
+you to resubmit your contributions to VOLK 3.0, which is LGPL licensed. And
+that's what we're asking you to do in this letter.
+
+Once your previous contributions are re-submitted, the copyright will fall back
+onto the original authors.
+
+##### What's going to happen to VOLK 2? Will there be other changes in VOLK 3.0 other than the license change?
+
+VOLK 2 will remain GPL-licensed. We may choose to continue supporting it with
+bug fixes, but the main development focus will lie on VOLK 3 going forward.
+
+The roadmap for VOLK 3.0 is not yet fully laid out, but we are considering
+making some API-breaking changes in VOLK 3 along with the relicensing.
+
+##### What are you asking from contributors?
+
+To grant us permission to use your contributions under the LGPL3 license for
+VOLK 3.0, we ask that you sign the paragraph HERE [ADD LINK]. You can do this either
+by submitting a pull request against THIS FILE [ADD LINK], adding your name and email
+address as they appear in previous commits. Alternatively, you can send us an
```

Comment:
- address
+ address(es)

as basically all major contributors have had multiple affiliations in the last couple years.